applications/dashboard/controllers/api/ThemesApiController.php

```diff
@@ -56,11 +56,20 @@ private function contentTypeByAsset(string $assetKey): string {
      * Get a theme assets.
      *
      * @param string $themeKey The unique theme key or theme ID.
+     * @param array $query
      * @return array
      */
-    public function get(string $themeKey): array {
+    public function get(string $themeKey, array $query): array {
```

Comment:
```suggestion
    public function get(string $themeKey, array $query = []): array {
```

I think the default is safer and may prevent some edge cases.